config/initializers/rollbar.rb

```diff
@@ -4,10 +4,14 @@
 
 Rollbar.configure do |config|
   config.access_token = Rails.application.secrets.rollbar_access_token
-
   config.enabled = false if Rails.env.development? || Rails.env.test?
-
   config.exception_level_filters.merge!("ActionController::InvalidCrossOriginRequest" => "ignore",
                                         "ActionController::RoutingError" => "ignore",
                                         "ActionController::UnknownFormat" => "ignore")
+  config.js_enabled = true
```

Comment:
It should be only enabled in staging or production, don't you think?

Reply:
Right. I've fixed it.